app/routes/api/stage-execution.js

```diff
@@ -96,7 +96,9 @@ export const stageExecutionHandlers = [{
         if (request.payload.claimOrApplication === 'claim') {
           const mainApplication = await getApplication(application.dataValues.applicationReference)
           sbi = mainApplication?.dataValues?.data?.organisation?.sbi
-          await updateApplicationByReference({ reference: request.payload.applicationReference, statusId, updatedBy: request.payload.executedBy, sbi })
+          // note that even though it using request.payload.applicationReference
+          // it can actually be the claim reference which is passed in the request
```

Comment:
This is confusing. We should probably improve this endpoint so that it makes more sense. Either rename the property to just `reference` or allow both application and claim references to be passed?

Reply:
It's in the back-office repo, thats the place that calls this endpoint. I dont disagree however, it should really be updated. I can create a ticket for us to tidy this up, this PR is mainly just to fix application to make it work again.

Reply:
yeah fair enough, outside the scope of this imediate bit of work, but something worth tidying up I think.